portal/portal-service-impl/impl/src/java/org/sakaiproject/portal/service/PortalServiceImpl.java

```diff
@@ -839,7 +840,7 @@ private boolean canUserUpdateSite(String userId, String siteId) {
 
 		try {
 			Site site = siteService.getSite(siteId);
-			return site.isAllowed(userId, SiteService.SECURE_UPDATE_SITE);
+			return ComponentManager.get(SecurityService.class).isSuperUser() || site.isAllowed(userId, SiteService.SECURE_UPDATE_SITE);
```

Comment:
Could you try defining the SecurityService on the class-level? Then it does not need to be retrieved every time the method is called and it can be reused in other methods.
Like:
```
@Autowired
private SecurityService securityService;
```
and

```suggestion
			return securityService.isSuperUser() || site.isAllowed(userId, SiteService.SECURE_UPDATE_SITE);
```

Reply:
@stetsche - That works for me. I'll force push an update to reflect that suggestion. Thanks.

Reply:
Nice!